packages/Responsive/media.js

```diff
@@ -0,0 +1,9 @@
+import breakpoints from './breakpoints'
+
+export default {
+  from: breakpoint => style => ({
+    [`@media (min-width: ${breakpoints[breakpoint]}px)`]: {
```

Comment:
Also, should this support `max-width` just like our `sass-mq` library did? Am I missing some other use case here?

Reply:
`min-width` should handle almost all cases right now. We can always enhance media to include `max-width` later.